headless-support-portal.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: 'Headless Support Portal'
 description: 'Give your customers visibility over their support requests.'
+sidebarTitle: 'Headless portal'
 ---
 
 <Warning>
```

Comment:
Just noticed that we don't link to the example repo in these docs anymore. Is that on purpose to push people to get in touch? If not then I think we should link to it somewhere in here

Reply:
Hmm that's a good point, let me add a link back in.

Reply:
Added!